package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -277,6 +277,23 @@ cyzone *externalRadius* *zMax* *zMin* *selection*
        relative to the COG of *selection*, instead of absolute z-values
        in the box.
 
+box *dimension(s)* *d1_Max* *d1_Min* (*d2_Max* *d2_Min*) (*d3_Max* *d3_Min*) *selection*
```

Comment:
I'm very likely missing something, but I don't know if I fully understand how this differs from using `prop` - could you maybe explain the benefits of this new selection over using a succession of `prop`?

Reply:
Well, `prop` is used to define a static zone in each axes, in my understanding. The new defined `box` selection do similar thing like which `sphzone`, `cyzone` or `cylayer` do, thus could define a zone according to another selection which could also be dynamical when `updating=True`.

Reply:
See @Cloudac7 's description in https://github.com/MDAnalysis/mdanalysis/issues/4323#issuecomment-1782184664 , which looks sensible to me. If we have cyzone then we can also have box.

Reply:
this would make more sense to me as `min` to `max` values rather than the opposite.  E.g. `box z 0 8 bilayer` to select a slice of water above a bilayer and `box z -8 0` to get the other half.

Reply:
@richardjgowers I had the same thoughts but unfortunately [cyzone and friends](https://docs.mdanalysis.org/stable/documentation_pages/selections.html#geometric) started this inverse "max min" syntax. I feel consistency across the `*zone` and `*layer` selections is more important here than purity. 

(I mean I am _really_ looking for someone to convince me we should do `min max` here ;-) because I know that I am going to mistype it...)